lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -3680,3 +3696,13 @@ dummyChangeAddressGen =
     ChangeAddressGen
         (maxLengthAddressFor (keyFlavor @s),)
         (maxLengthAddressFor (keyFlavor @s))
+
+utxoAssumptionsForWallet
+    :: forall s
+     . Excluding '[SharedKey] (KeyOf s)
```

Comment:
Request to use 4-space indents:

Either:
```suggestion
    :: forall s. Excluding '[SharedKey] (KeyOf s)
```
Or:
```suggestion
    :: forall s.
        Excluding '[SharedKey] (KeyOf s)
```

Reply:
Look, at the moment formatting follows the example of the `dummyChangeAddressGen` above. If we re-format this one then should we also re-format that one, to avoid inconsistency?

Reply:
>If we re-format this one then should we also re-format that one, to avoid inconsistency?

Yes please, if you're okay with doing it.

(Though please feel free to not do this if it makes more work!)

Reply:
Done